app/src/androidTest/java/org/mozilla/tv/firefox/ui/robots/NavigationOverlayRobot.kt

```diff
@@ -67,6 +67,10 @@ class NavigationOverlayRobot {
                 .check(matches(withHint(R.string.urlbar_hint)))
     }
 
+    fun waitForURLBarToDisplayHint() {
+        device.wait(Until.findObject(By.textContains("")), 5000)
```

Comment:
This seems misleading.  The current functionality looks more like `waitUntilAnyElementContainsEmptyText`.  Is there any way we can target this on the URL bar?